.github/stale.yml

```diff
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an Issue becomes stale
+daysUntilStale: 14
+# Number of days of inactivity before a stale Issue is closed
+daysUntilClose: 14
+# Only issues with all of these labels are checked if stale. Defaults to `[]` (disabled)
+onlyLabels:
+ - awaiting-user-response
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has no
+  recent activity. It will be closed if no further activity occurs. Thank you.
+# Comment to post when removing the stale label. Set to `false` to disable
+unmarkComment: false
```

Comment:
what does this do?

Reply:
if user responds  with in 14 days period , this will help remove the stale label .

Reply:
Do we need to create the "stale" label for this? We currently don't have one.

Reply:
Yes we need to create one stale label, thank you

Reply:
Created